### PR TITLE
[Tests-Only] Run trashbin tests on ocis

### PR DESCRIPTION
**Pull request by HariBhandari07**
## Description

Run trashbin webui tests on ocis
## Related Issue

- https://github.com/owncloud/product/issues/139
## How Has This Been Tested?

- https://github.com/owncloud/ocis/pull/475

## Screenshots (if appropriate):
https://github.com/owncloud/phoenix/pull/3982/files
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/features/webUITrashbin/trashbinDelete.feature [hunk 1]
```diff
@@ -33,12 +33,24 @@ Feature: files and folders can be deleted from the trashbin
...
     And folder "Folder,With,Comma" should not be listed on the webUI
 
+  @skipOnOC10
+  @issue-product-188
+  # after the issue is fixed delete this scenario and use the one above
+  Scenario: Delete folders and check that they are gone
+    When the user deletes folder "simple-folder" using the webUI
+    Then the error message with header "Deletion of simple-folder failed" should be displayed on the webUI
+    When the user deletes folder "Folder,With,Comma" using the webUI
+    Then folder "simple-folder" should be listed on the webUI
```
Comment (individual-it) [inline tests/acceptance/features/webUITrashbin/trashbinDelete.feature:51]: maybe swap that around to make it more logical

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/webUITrashbin/trashbinDelete.feature (hunk 1 above)
Comment (HariBhandari07) [reply tests/acceptance/features/webUITrashbin/trashbinDelete.feature]: done

**Comment by individual-it** [comment]
tests fail in CI (OCIS), maybe we need the newest commit-id of ocis in drone

**Comment by HariBhandari07** [comment]
> tests fail in CI (OCIS), maybe we need the newest commit-id of ocis in drone

done

**Review by individual-it — APPROVED** [review]

**Pull request merged**